### ISSUE TITLE
Remove Electron Framework.framework unnessary language iproj

### ISSUE THREAD

**Issue by Innei** [electron/packager#1762]
### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/packager/blob/main/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

### Problem Description

I want to delete unwanted language files. I found out how to delete the language file under Resource through this issue https://github.com/electron/forge/issues/2248. But I still don't know how to delete the language files of the framework. It is located under this location (macOS):

```
/Contents/Frameworks/Electron Framework.framework/Versions/A/Resources
```

I don't need to use these language files, but they take up ~40M in size.

### Proposed Solution

Provide a build parameter to set the language we need.

### Alternatives Considered

N/A

### Additional Information

Link https://github.com/electron/forge/issues/2248

**Comment by MarshallOfSound** [comment]
I don't think this should be a build parameter, but you can trivially delete this yourself in a hook
